src/fdk_rdf_parser/reference_data/extend_cpsvno_service.py

```diff
@@ -40,20 +40,18 @@ def extend_cv_contact_points(
 ) -> Optional[List[CVContactPoint]]:
     if contact_points is None:
         return contact_points
-    else:
-        extended: List[CVContactPoint] = list()
```

Comment:
Når du først er i gang, gidder du å fikse dette for hele `reference_data`-pakken?